scripts/upgrade_plugins.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/zsh
+
+## This script is to be run before a Packer release in order to update
+## all vendored plugins to the latest available release.
+## The SDK is included in the plugin list and will be upgraded as well if a
+## newest version is available.
+## This script should be run in packer's root.
+
+declare -a plugins=(
+    "alicloud"
```

Comment:
```suggestion
	"alicloud"
```
Just for indentation